src/workflows-app/components/SubmissionDetailsInputsTable.ts

```diff
@@ -96,6 +96,10 @@ const SubmissionDetailsInputsTable = ({ configuredInputDefinition }: SubmissionD
                         .source as RecordLookupInputSource;
                       return h(TextCell, [source.record_attribute]);
                     }
+                    if (inputTableData[rowIndex].source.type === 'literal') {
```

Comment:
I don't remember the exact schema definition, but will this also work for Optional literal inputs? If not I wonder if we should also add support for that too?

Reply:
Hm, that's a good question. I'll try to find a workflow with an optional to try and see how this works.

Reply:
Optional literal inputs work with this change as well
<img width="1512" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/68349264/46c105cb-4948-4249-b1f5-fefacf867d40">